segmentation/include/pcl/segmentation/impl/extract_polygonal_prism_data.hpp

```diff
@@ -243,7 +256,11 @@ pcl::ExtractPolygonalPrismData<PointT>::segment (pcl::PointIndices &output)
     pt_xy.x = pt[k1];
     pt_xy.y = pt[k2];
 
-    if (!pcl::isXYPointIn2DXYPolygon (pt_xy, polygon))
+    bool in_poly = false;
+    for (auto& poly : polygons)
+      in_poly ^= pcl::isXYPointIn2DXYPolygon(pt_xy, poly);
+
+    if (!in_poly)
```

Comment:
Please use {} even for single line loops/conditionals

Reply:
will do